tests/go_test.go

```diff
@@ -351,7 +351,7 @@ func CheckMutateBuffer(org []byte, offset flatbuffers.UOffsetT, fail func(string
 		testcase{"Pos.Y", func() bool { return monster.Pos(nil).MutateY(20.0) }},
 		testcase{"Pos.Z", func() bool { return monster.Pos(nil).MutateZ(30.0) }},
 		testcase{"Pos.Test1", func() bool { return monster.Pos(nil).MutateTest1(30.0) }},
-		testcase{"Pos.Test2", func() bool { return monster.Pos(nil).MutateTest2(20) }},
+		testcase{"Pos.Test2", func() bool { return monster.Pos(nil).MutateTest2(example.Color(20)) }},
```

Comment:
`20` is not a valid number in the schema, but we allow users to set it.
Would you update this to use one of the real enum values? e.g. `example.ColorBlue`

Reply:
Sure. This test originally used an "invalid" value so I just left it that way and added the cast. But a valid value is probably a better test.

I switched this to use a valid value and added another couple of tests for the invalid value, so both are exercised. Unfortunately there are no other enum fields present in the test object so I couldn't just use another one.